README.md

```diff
@@ -852,6 +852,150 @@ TODO-A: - TBD
 TODO-B: Problems: None
 ```
 
+
+# Re-Reviewing an approved package
+
+Right now this is **not active**, it is - so far - a documentation of the intent to do so helping all involved parties to better understand what this would mean.
+
+The MIR team and thereby in extension Ubuntu continuously evolves the MIR ruling, for example we add rules based on painful lessons learned to avoid the same to happen again. In that regard some asks for testing, quality and such might be best practise now, but have not been fulfilled by packages that are in main for a long time.
+
+The proposed approach would be to:
+
+1. The MIR team would aim for a constant demand of e.g. "1 review per week per MIR member"
+   1. On one hand that means if new requests are assigned in a week that we make no progress on the backlog, but OTOH this keeps effort plannable)
+2. We'd run a trimmed down check which is a subset of our usual check (see below)
+3. As on the initial MIR review we might end up recommending a security (re-)review which goes into their queue at low prio
+
+## Special cases
+Special case - packges in main without an launchpad bug based MIR process.
+Some older cases do not have anything for auditability, for those we'll reach out to the [owning team](http://reqorts.qa.ubuntu.com/reports/m-r-package-team-mapping.html) and ask to file an MIR as they should ask (and answer) themselfe the same questions we do on a modern MIR.
+
+## Progress
+
+We do not expect this to be quick, there are so many old packages in main that do not even have any trace how they got promoted.
+But if we never start it will never be better. So instead of holding back, we need to just start at some point.
+
+## Process
+
+We'd add (doing so is a TODO) a query that helps us to spot packages in main for the longest time without having a recent update on their MIR bugs - out of that pool we'd then assign in each weekly team meeting.
```

Comment:
```suggestion
We would add (doing so is a TODO) a query that helps us to spot packages in main for the longest time without having a recent update on their MIR bugs. Out of that pool we'd then assign re-reviews of approved packages in each weekly MIR team meeting.
```

suggestion: write out spoken English and explicitly name the implicit subject for better readability

Reply:
ack, done